x-pack/plugins/enterprise_search/public/applications/workplace_search/components/layout/personal_dashboard_layout/personal_dashboard_layout.scss

```diff
@@ -22,4 +22,8 @@
     width: 100%;
     height: 100%;
   }
+
+  .sourceStatus {
+    padding: 2px 0 0 4px;
+  }
```

Comment:
Since this is a personal dashboard layout file, I don't think these styles should be here.

You probably tried it, but want to make sure: is there a way to align everything with Eui Flexbox components instead?

Reply:
I could not figure it out with Flexbox only. [Here](https://codesandbox.io/s/lingering-cdn-jfi0i?file=/index.js) is a sandbox if you want to try. Not sure why the color is not showing up for the badge. You can still see the text is 1px to high on the right

Reply:
Thanks for the sandbox!

I don't really understand why, but looks like removing EuiText fixed the vertical alignment 🤷 

| Before | After |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/11838280/129949479-04311948-9d38-4da6-87d0-ac786fdbc680.png)  | ![image](https://user-images.githubusercontent.com/11838280/129949404-f569691d-cd78-4b04-844b-8e80219c48e5.png) |

Here's the code that I have:
```
{isFederatedSource && (
  <EuiFlexGroup gutterSize="none" justifyContent="flexEnd" alignItems="center">
    <EuiFlexItem grow={null}>
      <strong>{STATUS_LABEL}</strong>
    </EuiFlexItem>
    <EuiFlexItem grow={null}>
      <EuiHealth color="success">{READY_TEXT}</EuiHealth>
    </EuiFlexItem>
  </EuiFlexGroup>
)}
```

The space between "Status" and green circle could be increased by using `gutterSize="xs"`, but I'm not sure I like how it looks. The "Status:" part looks too separated from the rest of the content:

| gutterSize="none" | gutterSize="xs" |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/11838280/129949404-f569691d-cd78-4b04-844b-8e80219c48e5.png)  | ![image](https://user-images.githubusercontent.com/11838280/129950372-34a4c268-e98a-4a79-be2d-3724416f3a06.png) |

I'll leave it up to you though, it's not a strong opinion.

Reply:
That would great! Thanks!!